2021-finalproject/92495063/EnvironmentVCD.sh

```diff
@@ -0,0 +1,59 @@
+#!bin/bash
```

Comment:
Add screenshot for this script

Reply:
ok,l have add a screenshot 
missing-semester/2021-finalproject/92495063/EnvironmentCheck_Test.png

Reply:
Through your screenshot, your script didn't pass the last check item but print `All Check Passed`. Please fix that

Reply:
Now l update the script and screenshot